examples/blink/CMakeLists.txt

```diff
@@ -27,12 +27,17 @@ emil_build_for(hal_st.blink_nucleo64
     TARGET_MCU stm32g431
     TARGET_MCU stm32g474
     TARGET_MCU stm32wb55
+    TARGET_MCU stm32wba52
     PREREQUISITE_BOOL HALST_BUILD_EXAMPLES
 )
 
+
 target_compile_definitions(hal_st.blink_nucleo64 PUBLIC
-    $<$<NOT:$<STREQUAL:${TARGET_MCU},stm32wb55>>:NUCLEO=Nucleo64Ui>
+    $<JOIN:stm32g4x,stm32g431>
```

Comment:
I think something went wrong here. From the documentation:
$<JOIN:list,glue>
Joins the list with the content of the glue string inserted between each item